src/components/AppComponent.tsx

```diff
@@ -125,6 +126,7 @@ const AppComponent: FC = () => {
   const tutorial = useSelector(isTutorial)
   const splitPosition = useSelector(state => state.splitPosition)
 
+  WebviewBackground.changeBackgroundColor({ color: colors.bg })
```

Comment:
Should we put this in a `useEffect`? We usually want to avoid side effects directly in component render functions.

Reply:
Good call. Placed inside a `useEffect` here: 25268172daa0c7e55f005a587e9c803a5f2293d3